src/components/template/CreateTemplateWizard.tsx

```diff
@@ -60,6 +72,18 @@ export default function CreateTemplateWizard() {
         return (
           <TemplateReviewStep />
         );
+      case 5:
+        return successData
```

Comment:
why is it returning successData? when the responsibility of the renderStep is to return only the component?